boards/arm/qemu_cortex_m3/qemu_cortex_m3.yaml

```diff
@@ -5,5 +5,7 @@ arch: arm
 toolchain:
   - zephyr
   - gccarmemb
+supported:
+  - slip
```

Comment:
This sounds useless as we support "networking", the "slip" is just a bearer for transporting IP packets.